cancellation/workflow.go

```diff
@@ -31,11 +30,14 @@ func YourWorkflow(ctx workflow.Context) error {
 		if err != nil {
 			logger.Error("CleanupActivity failed", "Error", err)
 		}
+
 	}()
 
 	var result string
 	err := workflow.ExecuteActivity(ctx, a.ActivityToBeCanceled).Get(ctx, &result)
-	logger.Info(fmt.Sprintf("ActivityToBeCanceled returns %v, %v", result, err))
+	if err != nil {
+		return err
+	}
 
 	err = workflow.ExecuteActivity(ctx, a.ActivityToBeSkipped).Get(ctx, nil)
 	logger.Error("Error from ActivityToBeSkipped", "Error", err)
```

Comment:
The original purpose of this sample was to show that this is skipped because the context is already cancelled. Now that we return early unlike before, this is essentially unreachable code. Do you want to remove it?

Reply:
done, also added cleanup activity specific activity options (as it can fail heartbeat timeout, and its not heartbeating anyways)